package.json

```diff
@@ -31,6 +31,14 @@
     "prerelease": "npm run build",
     "release": "standard-version"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.js": "npm run lint"
+  },
```

Comment:
You had these in earlier PR too right ?

Reply:
Nope

Reply:
Ig you are talking about a similar PR to the `docsify` repo :thinking:

Reply:
ohh ok, my brain might be messing with me